### PR TITLE
Rearranged Hub owners header

### PR DESCRIPTION
**Pull request by MarvinBangert**
Rearranged "Hub owners" from the text of "IT Admins" to new line and bold the beginning

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/information-architecture-modern-experience.md [hunk 1]
```diff
@@ -62,8 +62,9 @@ Information architecture for your organization will be the most effective by col
...
+**IT Admins** – IT admins partner with organizational intranet owners and departmental business owners to implement high level navigational structure like implementing the [start page](https://support.microsoft.com/office/discover-content-with-the-sharepoint-start-page-6b85097a-87e0-4611-a29a-dfd49b1a1220) and [hubs](./create-hub-site.md). IT admins also help implement certain governance policies around [site creation and usage](./sites-usage-guidelines.md). IT admins are involved in planning, implementing, and maintaining information architecture as the business changes and scales.
+
+**Hub owners** – Hub owners manage hub-level content, branding, permissions, and navigational elements for hub in your organization’s intranet. Hub owners partner with departmental business owners and IT admins to [plan](./planning-hub-sites.md), create, and manage hubs throughout the lifecycle of your organization’s intranet architecture.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/information-architecture-modern-experience.md:67]:
```suggestion
**Hub owners** – Hub owners manage hub-level content, branding, permissions, and navigational elements for hubs in your organization’s intranet. Hub owners partner with departmental business owners and IT admins to [plan](./planning-hub-sites.md), create, and manage hubs throughout the lifecycle of your organization’s intranet architecture. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MarvinBangert, kindly commit the minor additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MattEEvans PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**